conans/client/tools/cmake.py

```diff
@@ -0,0 +1,11 @@
+from conans.client.tools.files import save
+
+
+def create_cmake_module_alias_targets(_, module, targets):
```

Comment:
Better to put it in the ``conan.tools.cmake`` space?

Reply:
it's already in `conan.client.tools.cmake` namespace